runtime/turing/src/lib.rs

```diff
@@ -1233,15 +1234,22 @@ impl_runtime_apis! {
 		fn fees(encoded_xt: Bytes) -> Result<Balance, Vec<u8>> {
 			let extrinsic: <Block as BlockT>::Extrinsic = Decode::decode(&mut &*encoded_xt).unwrap();
 			if let Call::AutomationTime(pallet_automation_time::Call::schedule_xcmp_task{
-				provided_id, execution_times, para_id, currency_id, encoded_call, encoded_call_weight
+				execution_times, para_id, currency_id, encoded_call_weight, ..
 			}) = extrinsic.clone().function {
 				let len = encoded_xt.len() as u32;
 
 				let xcm_fee = XcmpHandler::calculate_xcm_fee_and_weight(
```

Comment:
this fee also needs to be multiplied by `execution_times.len()`.

Reply:
I have another branch for resolving this. I'd prefer to release as separate PRs, but will leave that up to reviewers.